.github/workflows/on-push.yml

```diff
@@ -119,7 +119,11 @@ jobs:
       - name: Build NNTile native libraries
         run: |
           cmake -S . -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo -DUSE_CUDA=OFF
-          cmake --build build
+          cmake --build build -j
+      - name: Test CPU-only C++ tests
+        run: |
+          cp /usr/local/lib/libstarpu-*.so* /usr/lib
+          ctest --test-dir build -E wrappers -LE "(MPI|NotImplemented)" --output-on-failure
```

Comment:
I'd recommend to upload shared objects before testing. It could be useful for debugging certain issues. Also, it so's could be quite large so the better solution is to create symlinks or change install dir.